src/Ens/JobeetBundle/Resources/config/doctrine/Affiliate.orm.yml

```diff
@@ -0,0 +1,27 @@
+# src/Ens/JobeetBundle/Resources/config/doctrine/Affiliate.orm.yml
+Ens\JobeetBundle\Entity\Affiliate:
+  type: entity
+  table: affiliate
+  id:
+    id:
+      type: integer
+      generator: { strategy: AUTO }
+  fields:
+    url:
+      type: string
+      length: 255
+    email:
+      type: string
+      length: 255
+      unique: true
+    token:
+      type: string
+      length: 255
+    created_at:
+      type: datetime
+  oneToMany:
+    category_affiliates:
+      targetEntity: CategoryAffiliate
+      mappedBy: affiliate
+  lifecycleCallbacks:
+    prePersist: [ setCreatedAtValue ]
```

Comment:
@PhotonSapiens 
with Symfony / Doctrine there are many ways to define configuration
Although any of them is not "right" or "wrong" I ask you to **use annotations for ORM configuration** and **use .yml for service configuration**
Ask me how exactly do it in case of problems